dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -1,5 +1,6 @@
 from collections import defaultdict, Counter
 import itertools
+import string
```

Comment:
Should we maybe just `from string import punctuation` since that is all we need for this?

Reply:
I’m not sure about this, as we usually do tf.keras…. Or sklearn.metrics… or utils…